lib/pipelineTemplateVersionFactory.js

```diff
@@ -162,11 +163,25 @@ class PipelineTemplateVersionFactory extends BaseFactory {
             return [];
         }
 
-        return super.list({
+        const listConfig = {
             params: {
                 templateId: pipelineTemplateMeta.id
             }
-        });
+        };
+
+        if (config.sort) {
+            listConfig.sort = config.sort;
+        }
+
+        if (config.sortBy) {
+            listConfig.sortBy = config.sortBy;
+        }
+
+        if (config.paginate) {
+            listConfig.paginate = config.paginate;
+        }
+
+        return super.list(listConfig);
```

Comment:
we don't need to do this, we can pass the config directly

Reply:
we can't pass in config directly because it does not accept name & namespace